README.md

```diff
@@ -28,11 +28,18 @@ git clone https://github.com/SovereignCloudStack/rookify
 cd rookify
 ```
 
-3. __TODO:__ Install script
+3. For local installation of a development setup:
```

Comment:
I would write "To install the local development environment:"

Reply:
ok did that